lnrpc/routerrpc/router_server.go

```diff
@@ -760,23 +809,22 @@ func (s *Server) trackPayment(identifier lntypes.Hash,
 				continue
 			}
 
-			rpcPayment, err := router.MarshallPayment(result)
+			rpcPayment, err := s.cfg.RouterBackend.MarshallPayment(result)
```

Comment:
style nit: line too long (when using 8 spaces for the tab character).

Reply:
Done, I indeed had 4 spaces as tabsize...